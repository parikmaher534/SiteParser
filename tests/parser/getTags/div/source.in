<div class="test_parent">
	<header>
		<div class="inside_div">text</div>
		<div class="inside_div">text2</div>
		<div class="inside_div">text3</div>
		<div class="inside_div">
			text 2
			<div class="block_end"></div>
		</div>
	</header>
</div>
