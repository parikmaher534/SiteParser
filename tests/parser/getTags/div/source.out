<div class="test_parent">
<div class="inside_div">
<div class="inside_div">
<div class="inside_div">
<div class="inside_div">
<div class="block_end">
